components/match2/wikis/osu/get_match_group_copy_paste_wiki.lua

```diff
@@ -56,8 +56,8 @@ end
 ---@param bestof integer
 ---@return string[]?
 function WikiCopyPaste._getVetoes(args, bestof)
-	local vetoTypes = VETOES[bestof]
-	if not Logic.readBool(args.mapVeto) or not vetoTypes then return nil end
+	local vetoTypes = Array.copy(VETOES[bestof] or {})
+	if not Logic.readBool(args.mapVeto) or Logic.isEmpty(vetoTypes) then return nil end
 	if Logic.readBool(args.protect) then
 		table.insert(vetoTypes, 1, 'protect')
 	end
```

Comment:
```suggestion
	if not Logic.readBool(args.mapVeto) or not VETOES[bestof] then return nil end
	local vetoTypes = Array.extend(Logic.readBool(args.protect) and 'protect' or nil, args.mapVeto)
```

Reply:
`local vetoTypes = Array.extend(Logic.readBool(args.protect) and 'protect' or nil, args.mapVeto)`

If I replaced the old local vetoTypes line, this caused the generator to drop all the veto order and return the boolean `true` as text 
![image](https://github.com/user-attachments/assets/cc2748e4-c601-4120-9f79-9b2380c8ae15)

`if not Logic.readBool(args.mapVeto) or not VETOES[bestof] then return nil end` 
this line works however when replacing the old i`f not readBool` line

Reply:
Made a little typo, `args.mapVeto` at the end should be `VETOES[bestof]`